.gitignore

```diff
@@ -0,0 +1,86 @@
+
+# Everything in the external/packages directory as nuget packages are restored here.
+/nuget/restored_packages/*
+
+# Build generated files in src
+/src/**/objd/**/*
+/src/**/obj/**/*
```

Comment:
the gitignore needs to be updated as there are dlls in the PR, hashtag-counter/bin/Debug/netcoreapp3.1/Dapr.Actors.dll
Also there are files from obj/Debug in this PR

Reply:
Going to close this PR and create another, it seems it's easier to change in a fresh branch.